### DISCUSSION TITLE
ntptime

### DISCUSSION THREAD

**Discussion by kjm1102** [micropython/micropython#11079] [ESP32]
Curious how decisions are made re what to include in a port? eg ntptime, present in v1.17 is missing in v1.19. Is this by accident or design?

**Comment by bixb922** [comment]
I'm using ntptime on ESP32 with the 1.19.1 downloaded from https://micropython.org/download/esp32/ and see the module present:
```
MicroPython v1.19.1 on 2022-06-18; ESP32 module with ESP32
Type "help()" for more information.
>>> import ntptime
>>> dir(ntptime)
['__class__', '__name__', '__file__', 'socket', 'struct', 'time', 'settime', 'NTP_DELTA', 'host']
>>>
```
What Micropython download did you use?

**Comment by kjm1102** [comment]
Reloaded 1.19.1 & it's working now. I must have made a typo (import npttime or something?), my bad. Thnx for checking.
